diagrams/circle-example-euclidean.svg

```diff
@@ -402,7 +402,7 @@
       viewBox="0 -750 429 1000"
     >
       <g
-        stroke="none"
+        stroke="currentColor"
```

Comment:
@cmumatt Could you explain what this means?

Reply:
Yes, I also had to look into this last night, also -- thanks for asking about it.

Since we are now setting stroke at the path level (where it should be), we are no longer overwriting this DOM default, which is set when we instantiate the <g> node.

Reply:
OK! Thanks for clarifying.